Xamarin.Essentials/Screenshot/Screenshot.uwp.cs

```diff
@@ -21,7 +21,10 @@ static async Task<ScreenshotResult> PlatformCaptureAsync()
                 throw new InvalidOperationException("Unable to find main window content.");
 
             var bmp = new RenderTargetBitmap();
-            await bmp.RenderAsync(element).AsTask().ConfigureAwait(false);
+
+            // We must use ConfigureAwait(true), to ensure we keep the same execution context (thread); otherwise we'll get an exception
+            // in the ScreenshotResult constructor, when trying to access the properties of the RendertargetBitmap instance
+            await bmp.RenderAsync(element).AsTask().ConfigureAwait(true);
```

Comment:
We usually try to keep comments out of the code unless ncessary.

Couldn't we just remove ConfigurateAwait... as the default is true?

Reply:
Removing the ConfigureAwait makes sense and is more elegant. Thanks, will do that and also remove the comments (CET here, so will do it in a couple of hours...)

Reply:
Done!